lib/chat_web/live/main_live/page/room.ex

```diff
@@ -521,4 +519,26 @@ defmodule ChatWeb.MainLive.Page.Room do
     end)
     |> Kernel.not()
   end
+
+  def maybe_enable_usb_drive_dump(%{assigns: %{room: room}} = socket) when not is_nil(room) do
+    usb_drive_dump =
+      cond do
+        match?(%CargoRoom{status: :syncing}, socket.assigns[:cargo_room]) ->
+          :disabled
+
+        match?(
```

Comment:
When this is used ? why only `:dumping` ?

Reply:
User has the option to switch the room for sync and dump before the drive is plugged in, but we don't want to stop either cargo sync or drive dump while the drive is plugged in.